packages/web/src/lib/api.ts

```diff
@@ -3,16 +3,16 @@ import algosdk, { Algodv2, ALGORAND_MIN_TX_FEE, SuggestedParams } from "algosdk"
 import { ChainType, TxParams } from "../types";
 
 const mainNetClient = new Algodv2("", "https://algoexplorerapi.io", "");
-const testNetClient = new Algodv2("", "https://testnet.algoexplorerapi.io", "");
+const testNetClient = new Algodv2("", "https://node.testnet.algoexplorerapi.io", "");
```

Comment:
Could we use constant to store this URL? Btw can we import the URL from env?

Reply:
just a confirmation, we don't have an .env file right now, so you want me to create one and store it as a const in there and then import it from there. right?

Reply:
In my view, we should make testnetclient flexible. User can config this value(they can update/change it) . Not sure about use .env is best solution.

Reply:
Storing them in a constant file or env file is a good idea. Here is more info about it: https://medium.com/chingu/an-introduction-to-environment-variables-and-how-to-use-them-f602f66d15fa

Reply:
@vuvth why will the user change it .. ? I am not much familiar about it but I think it will always be the same host, as @rmeena840 also talked about creating an .env file, so how should I proceed?

Reply:
> @vuvth why will the user change it .. ? I am not much familiar about it but I think it will always be the same host

I think we shouldn't maintain this network URL. User should do that. I not sure this will change again or they will create multi test net in the next time. @Megha-Dev-19

Reply:
We should not have that global variable at all. Also no need to use `.env` - this should be avoided for libraries, this must work easily by just importing the library.

Instead, I propose to:
1. Define const values:
```
const mainnetAlgoExplorer = "https://algoexplorerapi.io";
const testnetAlgoExplorer = "https://node.testnet.algoexplorerapi.io"
```
2. rename `clientForChain` to `algoexplorerAlgod`